support/src/main/java/org/jenkinsci/plugins/workflow/support/steps/build/BuildTriggerStepExecution.java

```diff
@@ -78,6 +83,25 @@ public boolean start() throws Exception {
         }
     }
 
+    private void completeDefaultParameters(List<ParameterValue> parameters, Job<?,?> project) {
+        List<String> names = Lists.transform(parameters, new Function<ParameterValue, String>() {
+            @Override public String apply(ParameterValue input) {
+                return input.getName();
+            }
+        });
+        if (project != null) {
+            ParametersDefinitionProperty pdp = project.getProperty(ParametersDefinitionProperty.class);
+            if (pdp != null && pdp.getParameterDefinitionNames().size() > parameters.size()) {
+                for (String name : pdp.getParameterDefinitionNames()) {
+                    ParameterDefinition pDef = pdp.getParameterDefinition(name);
+                    if (!names.contains(pDef.getName()) && pDef.getDefaultParameterValue() != null) {
```

Comment:
:ant: call `getDefaultParameterValue()` only once (and only for an undefined parameter).